securedrop-admin

```diff
@@ -46,8 +46,21 @@ def activate_venv():
 def sdconfig(args):
     """Configure SD site settings"""
     activate_venv()
-    subprocess.check_call([os.path.join(ANSIBLE_PATH,'configure')],
-                            cwd=ANSIBLE_PATH, env=os.environ.copy())
+
+    ansible_cmd = [
+        'ansible-playbook',
+        # Passing an empty inventory file to override the automatic dynamic
+        # inventory script, which fails if no site vars are configured.
+        '-i', '/dev/null',
+        os.path.join(ANSIBLE_PATH, 'securedrop-configure.yml'),
+    ]
+    # Honor `--force` flag to clobber existing (misconfigured) vars,
+    # otherwise read in the existing vars to supply defaults.
+    if os.path.exists(SITE_CONFIG) and args.force is False:
+        ansible_cmd.append('--extra-vars')
+        ansible_cmd.append("@{}".format(SITE_CONFIG))
+
+    subprocess.check_call(ansible_cmd, cwd=ANSIBLE_PATH, env=os.environ.copy())
```

Comment:
Had thought it was necessary for the virtualenv to propagate, but minimal testing without it didn't break anything. So I'm inclined to think it's a no-op as well, as @dachary stated. Further testing required until I rip it out, though—thanks for review!